dataset/dataset-import.sh

```diff
@@ -52,7 +52,16 @@ dataset_provider_status () {
 
 dataset_provider_status_completed () {
   echo "Dataset provider status of the last completed task"
-  curl -ks "${KEYCLOAK_URI}status-completed"
+  t=0
+  until curl -ks "${KEYCLOAK_URI}status-completed" | grep '"success":"true"' ; do
+    sleep 1 && ((t=t+1))
+    echo "Polling...${t}s"
+  done
```

Comment:
Should this eventually time out (like after 2 minutes?). The timeout could be another CLI parameter.

Reply:
Ok, good idea on making that configurable, so that it could be used for larger datasets as well with a default value. Let me think about the change.

Reply:
Made this configurable via an optional argument, however it needs to be passed in a particular order, which I have mentioned in the help prompt